source/_integrations/tado.markdown

```diff
@@ -178,13 +178,13 @@ automation:
         - sensor.temp_sensor_room
         - sensor.tado_temperature
     
-    # Check if the room temp is more than 0.5 higher than the tado thermostat reading
+    # Check if the room temp is more than 0.5 away from the tado thermostat reading
     condition:
     - condition: template
       value_template: >
-        {% set tado_temp = states('sensor.tado_temperature')|float %}
-        {% set room_temp = states('sensor.temp_sensor_room')|float %}
-        {{ (tado_temp - room_temp) > 0.5 }}
+        {% set tado_temp = states('sensor.tado_temperature')|float(20) %}
+        {% set room_temp = states('sensor.temp_sensor_room')|float(20) %}
```

Comment:
Why default to 20?

Reply:
20 is the average room temperature in a heated room, at least in Celsius
land. Zero would result in a huge offset when one sensor goes wild.